File: ch13/Chapter13-CustomerConsumer-CS/App_Code/AdventureWorksDataService.cs
using System;
using System.Data.Services;
using System.Data.Services.Common;
using System.Collections.Generic;
using System.Linq;
using System.ServiceModel.Web;

public class AdventureWorksDataService : 
    DataService<AdventureWorksEntities>
{
    public static void 
        InitializeService(DataServiceConfiguration config)
    {
        config.SetEntitySetAccessRule("*", 
            EntitySetRights.AllRead);
    }
}

File: ch13/Chapter13-CustomerConsumer-VB/App_Code/AdventureWorksDataService.vb
Imports System.Data.Services
Imports System.Data.Services.Common
Imports System.Linq
Imports System.ServiceModel.Web

Public Class AdventureWorksDataService
    Inherits DataService(Of AdventureWorksEntities)

    Public Shared Sub InitializeService(ByVal config As  _
                                        DataServiceConfiguration)
        config.SetEntitySetAccessRule("*",
            EntitySetRights.AllRead)
    End Sub
End Class